va/validation-authority_test.go

```diff
@@ -213,12 +218,19 @@ func TestSimpleHttpTLS(t *testing.T) {
 	finChall, err := va.validateSimpleHTTP(ident, chall)
 	test.AssertEquals(t, finChall.Status, core.StatusValid)
 	test.AssertNotError(t, err, chall.Path)
+	logs := log.GetAllMatching(`^\[AUDIT\] Attempting to validate SimpleHTTPS for `)
+	test.AssertEquals(t, len(logs), 1)
+	test.AssertEquals(t, logs[0].Priority, syslog.LOG_NOTICE)
 }
 
 func TestSimpleHttp(t *testing.T) {
 	va := NewValidationAuthorityImpl(true)
 	va.DNSResolver = &mocks.MockDNS{}
 
+	log := mocks.NewSyslogWriter()
+	defer log.Close()
+	defer mocks.SwitchLog(mocks.SwitchLog(log))
+
```

Comment:
This usage is a bit awkward. Better to use `var log = UseMockLog()` (as in `web-front-end_test.go` below) in every `*_test.go` to divert all test logs to a mock regardless of whether they're being checked?